at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -77,7 +83,12 @@ class ScanVerbHandler extends AbstractVerbHandler {
             ? response.data?.split(',')
```

Comment:
logger.finer('response.data : ${response.data}'); Lets not have it in line 81

Reply:
Or may be have it with least level of logging and ideally servers should run with debug level